storage/rocksdb/ha_rocksdb_proto.h

```diff
@@ -71,6 +71,12 @@ Rdb_cf_manager &rdb_get_cf_manager();
 
 const rocksdb::BlockBasedTableOptions &rdb_get_table_options();
 bool rdb_is_ttl_enabled();
+bool rdb_is_ttl_read_filtering_enabled();
+#ifndef NDEBUG
```

Comment:
True. But we are already using `NDEBUG` in other areas of MyRocks.  Also RocksDB uses `NDEBUG` as well.  I think I'll keep this the same for now, we can always change later.